src/js/BookReader/utils.js

```diff
@@ -159,3 +159,19 @@ export function PolyfilledCustomEvent(eventName, {bubbles = false, cancelable =
   event.initCustomEvent(eventName, bubbles, cancelable, detail);
   return event;
 }
+
+/**
+ * Returns the number pixels something should be rendered at to be ~1n on the users
+ * screen when measured with a ruler.
+ */
+export function calcScreenDPI() {
```

Comment:
Q: should we unit test this also?

Reply:
I actually couldn't think of a way to unit test this one. Setting `width:1in` in jest seems to return an object with `width: 0px` 😬 .

Reply:
oh yes, i totally have encountered weird "stub dom" issues ^_^